cmd/rosetta/cli.go

```diff
@@ -120,6 +120,12 @@ VERSION:
 		Value: 50000,
 	}
 
+	cliFlagExtraGasLimitRelayedTxV3 = cli.UintFlag{
```

Comment:
Workaround, due to `MX-16423`. Did the same as for `cliFlagExtraGasLimitGuardedTx` (see code above).